stdlib/public/core/StringBridge.swift

```diff
@@ -250,11 +250,6 @@ public final class _NSContiguousString : _SwiftNativeNSString {
     }
   }
 
-  @objc
-  func _fastCharacterContents() -> UnsafeMutablePointer<UInt16>? {
-    return _core.elementWidth == 2 ? _core.startUTF16 : nil
-  }
-
   //
```

Comment:
I agree, but I don't know the history or reasoning behind why things are currently in the stdlib. Even if the type remains, all of the methods inside could be migrated over.

@dabrahams or @moiseev or @airspeedswift , do you know the reasoning here?